README.md

```diff
@@ -35,6 +35,10 @@ to the NumPyEigensolver. This may be useful for near-term quantum experiments, f
 that can still be solved classically, as their outcome can be easily compared against a classical
 equivalent since the same input data can be used._
 
+## Migration Guide
+
+Aqua has been deprecated. TBD - Link to Migration guide or more details here.
```

Comment:
Do you want to wait until this is here or merge this asap and add the link later?

Reply:
I don't know of any link currently to any migration guide. For release Pypi purposes the important is to have deprecated msg. The README.md can be changed at any time in master since the messages point to it there.

Reply:
Sounds good :+1: I just wanted to check whether the TBD needs to be addressed as part of this PR